mantidproject/mantid
# Remove QLines algorithm for BayesQuasi

Fixes #14951

BayesQuasi has replaced the QLines algorithm and is no longer required. Furthermore, the QLRun function in `IndirectBayes.py` has been removed as this functionality is now in the BayesQuasi algorithm. The system tests have been updated as required.
# To Test
- Ensure that QLines is no longer a subscribed algorithm and the source code has been removed
- Ensure that the system, unit and doc tests pass the build servers 
  - In particular ensure the `ISISIndirectBayesTest.py` system test runs successfully

**[Release Notes](http://www.mantidproject.org/Release_Notes_3_7_Indirect_Inelastic#Misc)**
